common/include/opae/mmio.h

```diff
@@ -138,24 +138,26 @@ fpga_result fpgaReadMMIO32(fpga_handle handle,
  * Map MMIO space
  *
  * This function will return a pointer to the specified MMIO space of the
- * target object in process virtual memory. Some MMIO spaces may be restricted
- * to privileged processes, depending on the used handle and type.
+ * target object in process virtual memory, if supported by the target. Some
+ * MMIO spaces may be restricted to privileged processes, depending on the used
+ * handle and type.
  *
  * After mapping the respective MMIO space, you can access it either through
```

Comment:
There is no "or" to the "either" here. I assume it was supposed to say "or the function calls", though worded better than that.

Also, the Doxygen for the fpgaReadMMIO32, etc. calls still says that fpgaMapMMIO() must be called first.